person.rb

```diff
@@ -36,12 +36,3 @@ def add_rental(book, date)
     Rental.new(book, date, self)
   end
 end
-
```

Comment:
- [ ] You did a great job for the most part. :+1: Currently, two major functionalities are throwing errors when trying to execute them. 🤝️
  - Listing People
  - Creating a rental
I believe this is happening because there were some issues when the associations between classes were set up. Listing people is crashing because you forgot to put a colon `(:)` when specifying an attribute accessor to the `name` variable. 😎️Creating a rental is not working because you don't have a rentals instance variable and an attribute accessor for it. Please do the following fixes so your app can run properly. 🥸️

| Before fix                                                             | After fix                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/Patfarmurs/oop-school-library/assets/17243325/b7025ad1-a870-4f73-986d-cd275a1f0811" width="300">|<img src="https://github.com/Patfarmurs/oop-school-library/assets/17243325/d4219d2d-b006-456a-b0aa-ca25a56f068d" width="300">|

Reply:
Thanks @amejid